projects/packages/forms/src/contact-form/class-contact-form-endpoint.php

```diff
@@ -40,7 +43,12 @@ class Contact_Form_Endpoint extends \WP_REST_Posts_Controller {
 			'settings_url' => 'admin.php?page=zerobscrm-plugin-settings',
 		),
 		'salesforce'                        => array(
-			'type'         => 'internal',
+			'type'         => 'service',
```

Comment:
The type for salesforce should always have been 'service'. Prior PR to add this didn't pick it up because were (very un-DRY-like) re-hard coding 'type' => 'service' in the get_service_status() callback. Now, for consistency, we always use the types defined here.